release-please-config.json

```
@@ -26,6 +26,9 @@
     "api": {
       "package-name": "api"
     },
+    "hapi-proto": {
+      "package-name": "hapi-proto"
```

Default is to release as `1.0.0`, let's pin this to `0.1.0` for now and then after the initial release we'll have to remove the pin and let it increment the minor version naturally
```suggestion
      "package-name": "hapi-proto",
      "release-as": "0.1.0"
```